lib/facilities/ppms_client.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module Facilities
+  # Core class responsible for api interface operations
+  class PPMSClient < Common::Client::Base
+    configuration Facilities::PPMSConfiguration
+
+    def provider_locator(params)
+      qparams = build_params(params)
+      response = perform(:get, 'v1.0/ProviderLocator?', qparams)
+      bbox_num = params[:bbox].map { |x| Float(x) }
+      response.body.select! do |provider|
+        provider['Latitude'] > bbox_num[1] && provider['Latitude'] < bbox_num[3] &&
+          provider['Longitude'] > bbox_num[0] && provider['Longitude'] < bbox_num[2]
+      end
+      response.body
```

Comment:
any consideration to ordering the results? would it make sense to sort here or leave it up to the caller and/or front end?

Reply:
The results are being sorted on the ppms side in order of driving distance to the provided address. To go with one of your other comments, I'll include a reference to the documentation in a comment to make these sorts of things clearer.